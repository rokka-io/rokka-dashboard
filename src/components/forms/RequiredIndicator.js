import React, { PropTypes } from 'react'

const RequiredIndicator = ({ required }) => required ? (<span> *</span>) : null
RequiredIndicator.propTypes = {
  required: PropTypes.bool
}

export default RequiredIndicator
