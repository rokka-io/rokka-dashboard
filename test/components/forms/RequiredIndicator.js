import React from 'react'
import RequiredIndicator from '../../../src/components/forms/RequiredIndicator'
import renderer from 'react-test-renderer'

test('RequiredIndicator does render', () => {
  const component = renderer.create(
    <RequiredIndicator />
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})

test('RequiredIndicator does render with required', () => {
  const component = renderer.create(
    <RequiredIndicator required />
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
