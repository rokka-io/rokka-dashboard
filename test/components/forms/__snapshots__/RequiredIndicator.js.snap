exports[`test RequiredIndicator does render 1`] = `null`;

exports[`test RequiredIndicator does render with required 1`] = `
<span>
   *
</span>
`;
